Cargo.toml

```diff
@@ -87,12 +87,22 @@ testdir = "0.7"
 data-encoding = "2.6.0"
 
 [features]
-default = ["net", "metrics", "engine", "rpc", "test-utils"]
+default = ["net", "metrics", "engine", "test-utils"]
 net = ["dep:iroh", "tokio/io-util", "dep:tokio-stream", "dep:tokio-util"]
 metrics = ["iroh-metrics/metrics", "iroh/metrics"]
 engine = ["net", "dep:iroh-gossip", "dep:iroh-blobs"]
 test-utils = ["iroh/test-utils"]
-cli = ["rpc", "dep:clap", "dep:indicatif", "dep:console", "dep:colored", "dep:dialoguer", "dep:shellexpand", "iroh-blobs/rpc", "iroh-base/base32"]
+cli = [
+  "rpc",
```

Comment:
Do we have any code left that exercises the cli stuff?

Reply:
ci tests with all features